content/en/tracing/_index.md

```diff
@@ -54,74 +60,83 @@ Now that you've configured your application to send traces to Datadog, start get
 
 ### Service Map
 
-[Understand service dependencies][2] with an auto-generated service map from your traces alongside service performance metrics and monitor alert statuses.
+[Understand service dependencies][4] with an auto-generated service map from your traces alongside service performance metrics and monitor alert statuses.
 
 {{< img src="tracing/index/ServiceMapInspect.gif" alt="Service Map"  style="width:100%;">}}
 
 ### Service Performance Dashboards
 
-[Monitor Service metrics][3] for requests, errors and latency percentiles.  Drill down into database queries or endpoints correlated with infrastructure.
+[Monitor Service metrics][5] for requests, errors and latency percentiles.  Drill down into database queries or endpoints correlated with infrastructure.
 
 {{< img src="tracing/index/ServicePage.gif" alt="Service Pages"  style="width:100%;">}}
 
-### Deployment Tracking
+### Live Search
 
-[Monitor service performance][4] and compare between versions for rolling, blue/green, shadow, or canary deployments.
+[Search 100% of your traces][6] by any tag live with no sampling for 15 minutes.
 
-{{< img src="tracing/deployment_tracking/ErrorRateByVersion.png" alt="Versions on the Service Page"  style="width:100%;">}}
+{{< img src="tracing/live_search/livesearchmain2.gif" alt="Live Search" >}}
 
-### Live Search
+### Live Analytics
+
+[Analyze performance by any tag on any span][7] live for 15 minutes during an outage to identify impacted users or transactions.
+
+{{< img src="tracing/live_search/LiveAnalytics2.gif" alt="Live Analytics" >}}
 
-[Search on any span][5] by any tag on 100% of your ingested traces live with no sampling for 15 minutes.
+### Trace Retention and Ingestion
 
-{{< img src="tracing/live_search/livesearchmain.gif" alt="Live Search" >}}
+[Retain the traces that matter most to you][8] with tag-based retention filters and perform analytics on all indexed spans for 15 days.
+
+{{< img src="tracing/index/RetentionFilterTracingPage.png" alt="Trace Retention and Ingestion"  style="width:100%;">}}
+
+### Deployment Tracking
+
+[Monitor service performance][9] and compare between versions for rolling, blue/green, shadow, or canary deployments.
+
+{{< img src="tracing/deployment_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}
```

Comment:
Did you mean to change the deployment tracking png name here?

Reply:
Yeah - trying out a different image - have a look and see if you think it's more compelling than what we have on prod